torchvision/prototype/datasets/decoder.py

```diff
@@ -0,0 +1,12 @@
+import io
+
+import numpy as np
+import PIL.Image
+import torch
+
+__all__ = ["pil"]
+
+
+def pil(file: io.IOBase, mode="RGB") -> torch.Tensor:
+    image = PIL.Image.open(file).convert(mode.upper())
```

Comment:
Do we want to close the file, or doesn't it make a difference? Maybe using a context manager here?

Reply:
IIRC, we don't need to. `.convert()` does this automatically.